payjoin-cli/src/app/v2/mod.rs

```diff
@@ -437,7 +437,7 @@ impl App {
         // Long poll until we get a response
         loop {
             let (req, ctx) = session.create_poll_request(
-                self.unwrap_relay_or_else_fetch(Some(session.endpoint().clone())).await?.as_str(),
+                self.unwrap_relay_or_else_fetch(Some(session.endpoint().into())).await?.as_str(),
```

Comment:
this would be alternatively `url::Url::parse(session.endpoint().as_str()).expect("Could not parse Url")` without the `into()` method on `inot_url::Url`